.github/workflows/test-npgsql.yml

```diff
@@ -63,35 +65,15 @@ jobs:
         with:
           dotnet-version: ${{ matrix.dotnet-version }}
 
-      - name: Use Npgsql version ${{ matrix.npgsql-version }}
-        run: |
-          sed -E 's!<PackageReference Include="Npgsql" Version=".+?" />!<PackageReference Include="Npgsql" Version="${{ matrix.npgsql-version }}" />!' \
-            -i demo.csproj
-
-      - name: Display .NET information
-        run: |
-          dotnet --version
-          dotnet --info
-
-      - name: Display package information
-        run: |
-          dotnet restore
-          dotnet list . package
-
-      - name: Invoke tests on .NET 5
-        if: matrix.dotnet-version == '5.0.x'
-        run: |
-          dotnet test --framework=net5.0 --collect:"XPlat Code Coverage"
```

Comment:
Apologies if I didn't understand correctly, do we loose now the tests with different platform versions?

Reply:
Nothing will be lost. All those stray commands have just been refactored into `testing/ngr.py`. It now has all the smartness "how to build for us" in one place.

Reply:
Yep, sorry, I missed: [dotnet_version_majorYep, sorry, I missed: dotnet_version_major](https://github.com/crate/cratedb-examples/pull/96/files#diff-75b364c2b98053dc2ce2711ebc3237f6e4e73ce920358bbd5e99fb37f8c58fcaR152)